views/register.ejs

```diff
@@ -1,27 +1,29 @@
 <%- include("partials/head.ejs") %>
 <%- include("partials/header.ejs") %>
-<form method="POST">
-  <div>
-    <label name="name">Enter your name:</label>
-    <input name="name" />
+<div class="d-flex flex-column align-items-center justify-content-center" style="width: 100%; height: 70vh;">
+  <div style="width: 30%; padding: 20px; background-color: #fff; box-shadow: 0px 2px 8px rgba(0, 0, 0, 0.1);">
+    <form method="POST">
+      <div class="form-group">
+        <label for="name">Enter your name:</label>
+        <input name="name" id="name" class="form-control" />
+      </div>
+      <div class="form-group">
+        <label for="email">Enter your email:</label>
+        <input name="email" id="email" class="form-control" />
+      </div>
+      <div class="form-group">
+        <label for="password">Enter your password:</label>
+        <input type="password" name="password" id="password" class="form-control" />
+      </div>
+      <div class="form-group">
+        <label for="password1">Confirm your password:</label>
+        <input type="password" name="password1" id="password1" class="form-control" />
+      </div>
+      <div>
+        <button class="btn mt-3" style="background-color: #d1e7dd; color: #0c4128; width: 100%;">Register</button>
```

Comment:
Nitpick:
All buttons should have `type=` attributes (in this case a `"submit"`).